src/shared/components/VideoPreview/VideoPreview.tsx

```diff
@@ -138,7 +138,7 @@ const VideoPreview: React.FC<VideoPreviewProps> = ({
 
   const metaNode = (
     <MetaText variant="subtitle2" main={main} scalingFactor={scalingFactor}>
-      {formatVideoViewsAndDate(views, createdAt, { fullViews: main })}
+      {views && formatVideoViewsAndDate(views, createdAt, { fullViews: main })}
```

Comment:
This condition can break it a bit, because date should still be displayed even when there's no views info. `formatVideoViewsAndDate` can handle `views` being `null` so we either need to make that `views || null` here or change the signature of `formatVideoViewsAndDate` to accept `undefined`